src/appleseed.shaders/include/alshaders/noise.h

```diff
@@ -0,0 +1,367 @@
+
```

Comment:
@est77 i'll be moving this to the AS noise helpers headers since it'll be used by the asNoise custom node as well (for the crocodile noise, advection noise, etc).